src/modules/Hosts/Hosts/Strings/en-us/Resources.resw

```diff
@@ -299,6 +299,12 @@
     <value>Entries contain too many hosts</value>
     <comment>"Hosts" refers to the system hosts file, do not loc</comment>
   </data>
+  <data name="EntryAddressIsInvalidWarning.Text" xml:space="preserve">
+    <value>Address not recognized as IPv4 or IPv6</value>
```

Comment:
```suggestion
    <value>Must be a valid IPv4 or IPv6 address</value>
```

Reply:
Grammatically correct I think: `Has to be a valid ....`